auth/jwt.go

```diff
@@ -111,7 +111,8 @@ func (id *jwtIdentity) Valid() error {
 	}
 
 	now := jwt.TimeFunc().UTC().Unix()
-	if now < id.IssuedAt {
+	// provide tolerance for clockdrifting slower
+	if now < id.IssuedAt-int64(ClockDriftDelta.Seconds()) {
```

Comment:
3 Questions:
1. Does this ever get called?
2. If it does, should we use the ClockDriftDelta here or the ExpirationDelta?  This is checking the token's expiration, so I think we should use ExpirationDelta
3. I think we should use the same tolerance in the other direction for the id.Expired() method below